tests/object/test.bats

```diff
@@ -0,0 +1,84 @@
+# PURPOSE:
+# I will test the following commands: object create, get, list, update, url, delete, and clean.
+load ../test_setup.bash
+
+# constants
+obj1="test1.txt"
+obj2="test2.txt"
+obj3="test3.txt"
+
+@test "object create" { # object create test
+  
+  filePath="$(pwd)/tests/object/objectTesting/""${obj1}" # path to test1.txt
+
+  run $NIM object create "$filePath"
+  assert_success
+  assert_output --partial "done"
+
+	
+}
+
+@test "object get" { # object get test
+
+  run $NIM object get $obj1
```

Comment:
Can you check the contents of the file match the expected value?

Reply:
Yes, I can check and make sure in the test.